sysstat fails build on 1-core VMs

While working on [reproducible builds](https://reproducible-builds.org/) for [openSUSE](https://en.opensuse.org/openSUSE:Reproducible_Builds), I found that
our `sysstat` 12.6.0 package consistently fails to build on 1-core VMs.

This is how we build:
https://code.opensuse.org/package/sysstat/blob/master/f/sysstat.spec#_82

on Debian and openSUSE, it can be reproduced with
```
osc checkout openSUSE:Factory/sysstat && cd $_
osc build --vm-type=kvm -j1 --clean --noservice standard
```

build log:
```
[   13s] gcc -o sadc -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -Wall -Wstrict-prototypes -pipe -O2 sadc.o act_sadc.o sa_wrap.o sa_common_light.o common_light.o systest.o librdstats.a librdsensors.a -L. -lsyscom -lsensors
[   13s] /usr/lib64/gcc/x86_64-suse-linux/12/../../../../x86_64-suse-linux/bin/ld: cannot find -lsyscom: No such file or directory
[   13s] collect2: error: ld returned 1 exit status
[   13s] make: *** [Makefile:251: sadc] Error 1
```

This typically is a result of under-specified build dependencies in Makefiles.

--- a/Makefile.in
+++ b/Makefile.in
@@ -338,7 +338,7 @@ sadc.o: sadc.c sa.h version.h common.h rd_stats.h rd_sensors.h
 
 sadc: LFLAGS += $(LFSENSORS)
 
-sadc: sadc.o act_sadc.o sa_wrap.o sa_common_light.o common_light.o systest.o librdstats.a librdsensors.a
+sadc: sadc.o act_sadc.o sa_wrap.o sa_common_light.o common_light.o systest.o librdstats.a librdsensors.a libsyscom.a
 
 sar.o: sar.c sa.h version.h common.h rd_stats.h rd_sensors.h
 

